### PR TITLE
readme in tr folders with link to view as web page

### PR DESCRIPTION
**Pull request by PeterConstable**
Would this be a useful convenience to add (making it easy to get from the source to view the page as a web page)?

add tr9 readme with link to view as web page (example)

### CODE REVIEW

File: tr9/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1 @@
+To view as HTML, use this link: https://unicode-org.github.io/unicode-reports/tr9/tr9.html
```
Comment (srl295) [inline tr9/ReadMe.md:1]:
```suggestion
To view as HTML, use this link: <https://unicode-org.github.io/unicode-reports/tr9/tr9.html>
```

correcting bare URL to MD

**Comment by srl295** [comment]
Makes sense. It could be generated by a script, which could also keep a top-level index up to date.

**Comment by macchiati** [comment]
That would be a handy feature.

Mark

**Comment by srl295** [comment]
> O.k. with me, I guess. But seems overkill to have to add a ReadMe like this for _every_ reports directory. This information would be more useful to editors simply provided as a table of review links for reports accessible off the edcom page.

OK, so just having a top level link to every report.  In that case, rather than having lots of small files, we can just modify the top level Readme.md visible at  <https://unicode-org.github.io/unicode-reports/>  to have a list of _relative_ links into each TR.

**Comment by PeterConstable** [comment]
I replaced the per-folder readme with an update to the root folder readme to add links.

Note: the root readme had a link suggesting you could browse the repo contents as web pages. But that didn't actually work: it only allowed you to view the readme as a web page; you couldn't browse from there to the pages in the subfolders.

**Comment by srl295** [comment]
postscript: In https://github.com/unicode-org/unicode-reports/commit/5290334d1b531e66d6062ecb3ad777407c383057 I added a dynamic indexer.

**Comment by macchiati** [comment]
Nice

Mark

**Pull request merged**